dApp/user/src/Apollo.res

```diff
@@ -2,12 +2,43 @@ let graphqlEndpoint = "localhost:8080/v1/graphql"
 
 let headers = {"x-hasura-admin-secret": "testing"}
 
+
+type clientHeaders = {
+    @as("eth-address")
+    ethAddress: string,
+    @as("eth-signature")
+    ethSignature: string,
+}
+
+let getAuthHeaders = (~user) => {
+  open Ethers.Utils
+    switch(user){
+      | None => None
+      | Some(u) => {
+          let getUserSignature = Dom.Storage2.getItem(_, u->ethAdrToLowerStr)
+          switch(getUserSignature(Dom.Storage2.localStorage)){
```

Comment:
Ahh, cool, I wrote my own bindings to this last time, convenient that it already exists :)